Doc comma-separated lists of operator versions (#7877)
open-cluster-management/rhacm-docs: governance/operator_pol_ctrl.adoc

| `spec.versions`
| Optional

> | Declare which versions of the operator are compliant. The `versions` parameter is a list of templatable strings that specifies which installed `ClusterServiceVersion` names are compliant when in `inform` mode, and which `InstallPlans` are approved when in `enforce` mode. Multiple versions can be provided in one entry by separating them with commas. If the field is empty the strings are ignored and all `ClusterServiceVersion` names are approved. The default value is empty.

Reviewer: @JustinKuli thanks so much! What do you think of this rewrite to make it more user focused? 

```suggestion
| Declare which versions of the operator are compliant. The `versions` parameter is a list of templatable strings that specifies which installed `ClusterServiceVersion` names are compliant when in `inform` mode, and which `InstallPlans` are approved when in `enforce` mode. To provide different versions in one entry, separate the versions with commas. If the field is empty, the strings are ignored and all `ClusterServiceVersion` names are approved. The default value is empty.
```

Author: Added! :+1:

> | Declare which versions of the operator are compliant. The `versions` parameter is a list of templatable strings that specifies which installed `ClusterServiceVersion` names are compliant when in `inform` mode, and which `InstallPlans` are approved when in `enforce` mode. To provide different versions in one entry, separate the versions with commas. If the field is empty, the strings are ignored and all `ClusterServiceVersion` names are approved. The default value is empty.

Reviewer: Very minor, but we're encouraged to use "give" instead of "provide" 
```suggestion
| Declare which versions of the operator are compliant. The `versions` parameter is a list of templatable strings that specifies which installed `ClusterServiceVersion` names are compliant when in `inform` mode, and which `InstallPlans` are approved when in `enforce` mode. To give different versions in one entry, separate the versions with commas. If the field is empty, the strings are ignored and all `ClusterServiceVersion` names are approved. The default value is empty.
```
